docs/self-managed/setup/deploy/local/docker-compose.md

```diff
@@ -1,26 +1,212 @@
 ---
 id: docker-compose
 title: "Docker Compose"
-keywords: ["camunda docker"]
+keywords: ["camunda docker", "docker compose"]
 ---
 
-A Docker Compose configuration to run Zeebe, Operate, Tasklist, Optimize, Identity, and Connectors Bundle is available in the [camunda-platform](https://github.com/camunda/camunda-platform/blob/main/docker-compose.yaml) repository.
-Follow the instructions in the [README](https://github.com/camunda/camunda-platform#using-docker-compose).
+A Docker Compose configuration to run Camunda Self-Managed components (e.g., Zeebe, Operate, Tasklist, Optimize, Identity, and Connectors Bundle, etc.).
 
-:::warning
+:::danger
 While the [Docker images](/self-managed/setup/deploy/other/docker.md) themselves are supported for production usage, the Docker Compose files are designed to be used by developers to run an environment locally; they are not designed to be used in production. We recommend to use [Kubernetes](/self-managed/setup/install.md) in production.
 :::
 
-This Docker Compose configuration serves two purposes:
+## Prerequisites
+
+- **Docker Compose:** Version 1.27.0+ (supports the [latest compose specification](https://docs.docker.com/compose/compose-file/)).
+- **Docker:** Version 20.10.16+.
+- **Local:** Add Keycloak to resolve to 127.0.0.1 on your local machine, and set `KEYCLOAK_HOST=keycloak` in the `.env` file for token refresh and logout functionality.
+
+## Setup
+
+### Start
+
+To spin up a Camunda Platform 8 Self-Managed environment locally you can use the following docker compose configuration files:
+
+- **docker-compose.yaml:** Contians all Camunda 8 Components for a full stack deployment: Zeebe, Operate, Tasklist, Connectors, Optimize, Identity, Elasticsearch, Keycloak, Web Modeler, and PostgreSQL.
+- **docker-compose-core.yaml:** Contians Camunda 8 Orchestration cluster components: Zeebe, Tasklist, Operate, Optimize, Identity and Connectors.
+- **docker-compose-web-modeler.yaml:** Contians Camunda 8 Web Modeler standalone installation.
+
+To start a complete Camunda Platform 8 Self-Managed environment locally:
+
+1. Clone this repository.
```

Comment:
This should use the new repo at https://github.com/camunda/camunda-self-managed?

Reply:
Fixed :+1: 
Also, we will move away from that style (cloning) and provide artifacts to be downloaded directly.
But that's not here yet.